### PR TITLE
Add Fody.ConfigureAwait

### PR DESCRIPTION
**Pull request by willsb**
Making our lives a lot simpler

### CODE REVIEW

**Comment by amulware** [comment]
This seems like it might be confusing for new developers. I.e. they would not know that we are using it and might get confused why we never call the method in code, or they might call the method themselves. We can fill them in easily enough, and code review will prevent silly things, but are we sure it's worth it? It sort of means we will worry less about whether to await here or there, and might start doing it wrong in other places?

TL;DR: I'm worried that doing things like this implicitly will make us forget to do it where we should explicitly.

Thoughts?

**Comment by willsb** [comment]
This is basically changing the default behavior from `ConfigureAwait(true)` to `ConfigureAwait(false)`, and this default makes a lot of sense in this lib. I don't think this will make us lower our guard in any way, specially because I just want this to be active in Ultrawave and PrimeRadiant
